envisage/extension_point.py

```diff
@@ -141,6 +141,12 @@ def __init__(self, trait_type=List, id=None, **metadata):
 
         return
 
+    def __repr__(self):
+        """ String representation of an ExtensionPoint object """
+        return "ExtensionPoint(id={})".format(self.id)
+
+    __str__ = __repr__
```

Comment:
This is not needed.

Reply:
addressed in commit https://github.com/enthought/envisage/pull/142/commits/280f6f5b063ae664cf9d4ad48dd76c606ebc220e